.drone.star

```diff
@@ -2534,40 +2502,17 @@ def cacheOcis():
         ],
     }]
 
-def stopBuild():
-    return [{
-        "name": "stop-build",
-        "image": DRONE_CLI_ALPINE,
-        "environment": {
-            "DRONE_SERVER": "https://drone.owncloud.com",
-            "DRONE_TOKEN": {
-                "from_secret": "drone_token",
-            },
-        },
-        "commands": [
-            "drone build stop owncloud/web ${DRONE_BUILD_NUMBER}",
-        ],
-        "when": {
-            "status": [
-                "failure",
-            ],
-            "event": [
-                "pull_request",
-            ],
-        },
-    }]
-
 def uploadScreenshots():
     return [{
         "name": "upload-screenshots",
         "image": PLUGINS_S3,
         "pull": "if-not-exists",
         "settings": {
             "bucket": {
-                "from_secret": "cache_public_s3_bucket",
+                "from_secret": "cache_s3_bucket",
             },
             "endpoint": {
-                "from_secret": "cache_public_s3_server",
+                "from_secret": "cache_s3_server",
```

Comment:
`cache_public_s3_server` can be left unchanged
```suggestion
                "from_secret": "cache_public_s3_server",
```

Reply:
Don't mix it. Use consistent secrets `cache_public_s3_*` _or_ `cache_s3_*`